src/components/GeneralInfo/SystemCard/SystemCard.js

```diff
@@ -61,16 +68,190 @@ class SystemCardCore extends Component {
       hasDisplayName,
       hasAnsibleHostname,
       hasWorkspace,
-      hasSAP,
       hasSystemPurpose,
       hasCPUs,
       hasSockets,
       hasCores,
       hasCPUFlags,
       hasRAM,
       extra,
+      workloadsData,
     } = this.props;
     const { isDisplayNameModalOpen, isAnsibleHostModalOpen } = this.state;
+    const workloadsTypesKeys = [
+      'ansible',
+      'crowdstrike',
+      'ibm_db2',
+      'intersystems',
+      'mssql',
+      'oracle_db',
+      'rhel_ai',
+      'sap',
+    ];
+    function checkWorkloadsKeys(input = {}, referenceKeys) {
+      return referenceKeys.filter(
+        (key) => typeof input[key] === 'object' && input[key] !== null
+      );
+    }
+    const workloadsTypes = checkWorkloadsKeys(
+      workloadsData,
+      workloadsTypesKeys
+    );
+
+    const workloadConfigs = [
```

Comment:
Let's extract this to a constant (like, WORKLOAD_CONFIGS) outside of this component. This should not be recreated with every component render

Reply:
Did that :)